src/metrics.ts

```diff
@@ -121,6 +129,9 @@ export default class Metrics {
             [this.headerName]: this.clientKey,
             Accept: 'application/json',
             'Content-Type': 'application/json',
+            'x-unleash-sdk': `unleash-js@${this.sdkVersion}`,
```

Comment:
we can start with just unleash-js even when used in other SDKs like React or Vue. Over time we can make it configurable if we have a good use case for it.

Reply:
Yeah, cool. I ilke that.